### PR TITLE
update to qenv

### PR DESCRIPTION
**Pull request by nikolas-burkoff**
Part of https://github.com/insightsengineering/teal.code/issues/34 see https://github.com/insightsengineering/teal.code/pull/74

I also replaced dontrun with if (interactive()) as it was annoying me so much not being able to run the apps in RSrudio easily

### CODE REVIEW

File: R/tm_file_viewer.R [hunk 1]
```diff
@@ -114,6 +114,7 @@ ui_viewer <- function(id, ...) {
 }
 
 srv_viewer <- function(id, input_path) {
+  checkmate::assert_class(data, "tdata")
```
Comment (Polkas) [inline R/tm_file_viewer.R:117]:
```suggestion
```

**Comment by Polkas** [comment]
LGTM - approve

File: R/tm_front_page.R [hunk 2]
```diff
@@ -77,7 +77,7 @@ tm_front_page <- function(label = "Front page",
     ui = ui_front_page,
     ui_args = args,
     server_args = list(tables = tables, show_metadata = show_metadata),
-    filters = NULL
+    filters = if (show_metadata) "all" else NULL
```
Comment (nikolas-burkoff) [inline R/tm_front_page.R:80]: I don't like this, but if I don't do this I get an error from teal saying if filters = NULL then you cannot use data

**Review by mhallal1 — APPROVED** [review]
LGTM

**Pull request merged**